deploy/docker/Dockerfile

```diff
@@ -1,13 +1,17 @@
 # Docker image for the primary terria map application server
-FROM node:8
+FROM osgeo/gdal:alpine-small-3.1.0 as build
 
-RUN apt-get update && apt-get install -y gdal-bin
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash git openssh npm
+RUN mkdir -p /etc/config/client
+WORKDIR /usr/src/app
 
-RUN mkdir -p /usr/src/app && mkdir -p /etc/config/client
-WORKDIR /usr/src/app/component
-COPY . /usr/src/app
-RUN rm wwwroot/config.json && ln -s /etc/config/client/config.json wwwroot/config.json
+COPY ./package.json ./
+RUN npm install
 
-EXPOSE 3001
+COPY . .
+
+RUN npm run gulp
+
+RUN ln -s /etc/config/client/config.json wwwroot/config.json
 
-CMD [ "node", "./node_modules/terriajs-server/lib/app.js", "--config-file", "devserverconfig.json" ]
```

Comment:
Why don't we want Terria to be launched along with the container? I think some of our deployment systems for some of our maps assume that this happens, and this could break things

Reply:
You are right, this could break one of your deployment pipelines. I removed it because we prefer to keep the container generic and then decide the command to run at run time, the command we run is defined in our Helm chart

Reply:
And then I added a readme that explains to users how to use it